es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = number + street;
+
+    console.log(`LearningFuze: ${address}`);
```

Comment:
Please sure you are A) converting _all_ concatenation operations to template literals, and B) not modifying other parts of the code.

---

![image](https://user-images.githubusercontent.com/16143771/127206909-d3b4cd4d-e37b-4400-9886-6ece7656a6ed.png)

---

![image](https://user-images.githubusercontent.com/16143771/127206928-a3b3b6ee-cad8-4dea-8493-eb37c08be8ad.png)